src/remote/remote.h

```diff
@@ -192,23 +193,42 @@ struct Rdb : public Firebird::GlobalStorage, public TypedHandle<rem_type_rdb>
 };
 
 
+const SQUAD NULL_BLOB = {0, 0};
+
+inline bool operator==(const SQUAD& s1, const SQUAD& s2)
+{
+	return s1.gds_quad_high == s2.gds_quad_high &&
+		s2.gds_quad_low == s1.gds_quad_low;
+}
+
+inline bool operator>(const SQUAD& s1, const SQUAD& s2)
+{
+	return s1.gds_quad_high > s2.gds_quad_high ||
```

Comment:
For readability, please use parenthesis when mixing different logical operators.

Reply:
Ok